rest/admin_api.go

```diff
@@ -1439,6 +1450,16 @@ func (h *handler) handleSGCollect() error {
 
 	h.writeRawJSONStatus(http.StatusOK, []byte(`{"status":"started"}`))
 
+	auditFields := base.AuditFields{
+		"output_dir":   params.OutputDirectory,
+		"upload_host":  params.UploadHost,
+		"customer":     params.Customer,
+		"ticket":       params.Ticket,
+		"keep_zip":     params.KeepZip,
+		"zip_filename": zipFilename,
+	}
+	base.Audit(h.ctx(), base.AuditIDSyncGatewayCollectInfoStop, auditFields)
```

Comment:
This events should have `mandatoryFieldGroups: []fieldGroup{fieldGroupRequest, fieldGroupAuthenticated}`